product_docs/docs/pem/9/reports.mdx

```diff
@@ -22,9 +23,21 @@ Only superusers or the users with the pem_admin role can download the System Con
 
 Information in these reports shows the latest probe run time.
 
-## System Configuration Report
+## Alert History report
 
-The System Configuration Report provides detailed information about the PEM Agents group, PEM Server Directory group, and custom groups listed under the browser tree. These groups can contain Postgres Enterprise Manager, PEM agent, and database servers. You can download this report in HTML and JSON formats.
+The Alert History report provides detailed information about the alerts history at the agent or server level in JSON format.
+
+Select **Management > Reports > Alert History Report**. From the dialog box, select your options:
+
+- **Agent/Server Name** &mdash; Select the agents and servers from the list. The **Overall System Report** option is also available to generate a report for all the registered and active agents and servers.
+
+- **Timeframe** &mdash; Select the required timeframe from the list.
+
+- **Alert Types** &mdash; Select the alert types you want to generate the alert history report for.
+
+## System Configuration report
+
+The System Configuration report provides detailed information about the PEM Agents group, PEM Server Directory group, and custom groups listed under the browser tree. These groups can contain Postgres Enterprise Manager, PEM agent, and database servers. You can download this report in HTML and JSON formats.
```

Comment:
Is Postgres Enterprise Manager supposed to be PEM server in the second sentence?